_targets.R

```diff
@@ -330,11 +366,14 @@ list(
     rnet_cl = rnet_commute_list
     rnet_sl_p = rnet_school_list$Primary
     rnet_sl_s = rnet_school_list$Secondary
+    rnet_quietness = rnet_segments
     names(rnet_cl) = paste0("commute_", names(rnet_cl))
     names(rnet_sl_p) = paste0("primary_", names(rnet_sl_p))
     names(rnet_sl_s) = paste0("secondary_", names(rnet_sl_s))
     
-    rnet_combined = combine_rnets(rnl = c(rnet_cl, rnet_sl_p, rnet_sl_s),
+    rnl = c(rnet_cl, rnet_sl_p, rnet_sl_s, list(rnet_quietness))
+    
+    rnet_combined = combine_rnets(rnl = rnl,
```

Comment:
This stage seems to be taking forever, more than an hour, on my laptop:

```
▶ start target combined_network
2023-08-15 05:59:32.987183 constructing segments
2023-08-15 06:18:38.173306 building geometry
2023-08-15 06:19:31.34229 simplifying geometry
large data detected, using regionalisation, nrow = 1802185
2023-08-15 06:22:13.528412 regionalisation complete, aggregating flows
```

Reply:
I only see 24 minutes in the times? Whats the progress on the last stage?

Reply:
It had hanged for ~60 minutes by the time I posted that message. Key question: can you reproduce this behaviour?

Reply:
Can you share the routes? Would be good to have a large scale test dataset on teams.

Reply:
> Can you share the routes? Would be good to have a large scale test dataset on teams.

Sure but not on the home laptop ATM. Just run `tar_make()` and wait a few minutes on this branch.